Makefile.docs

```diff
@@ -0,0 +1,90 @@
+# user defined variables
+LANGUAGES ?=
+
+# python specific variables
+PYTHON_MAIN_DIR ?= src
+
+# provided by docsrv
+DESTINATION_FOLDER ?=
+SHARED_FOLDER ?=
+BASE_URL ?=
+REPOSITORY_OWNER ?=
+REPOSITORY ?=
+VERSION_NAME ?=
+
+API_DOCS_FOLDER := "$(DESTINATION_FOLDER)/api"
+CI_FOLDER := "$(SHARED_FOLDER)/ci"
+
+docs: docs-site docs-api
+
+install-ci:
+	@if [ ! -d "$(CI_FOLDER)" ]; then \
+		git clone https://github.com/src-d/ci $(CI_FOLDER); \
```

Comment:
This is the CI repo... in which situation is executed this `make` rule without the repo?
If the intend is to put it into `$(SHARED_FOLDER)/ci`, why not just copying it from `.` to that folder